db/data/20221130071823_move_bd_patients_to_different_facility.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class MoveBdPatientsToDifferentFacility < ActiveRecord::Migration[6.1]
+  def up
+    return unless CountryConfig.current_country?("Bangladesh") && SimpleServer.env.production?
+
+    Patient.where(
+      registration_facility_id: "fe48375c-7826-41dd-9110-d716a9181e8f",
+      registration_user_id: "ced029f9-4e06-40cd-b719-605c85a4b004"
+    )
+      .update_all(
+        registration_facility_id: "daff41a3-4922-41b0-a822-6fef2db07e68",
+        assigned_facility_id: "daff41a3-4922-41b0-a822-6fef2db07e68"
```

Comment:
Let's not update the `assigned_facility_id` for all patients since they might have been legitimately assigned to another facility. 

In this case there is just one [patient](https://metabase.bd.simple.org/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7InR5cGUiOiJxdWVyeSIsInF1ZXJ5Ijp7InNvdXJjZS10YWJsZSI6MTksImZpbHRlciI6WyJhbmQiLFsiPSIsWyJmaWVsZCIsMzc0LG51bGxdLCJmZTQ4Mzc1Yy03ODI2LTQxZGQtOTExMC1kNzE2YTkxODFlOGYiXSxbIj0iLFsiZmllbGQiLDM2OSxudWxsXSwiY2VkMDI5ZjktNGUwNi00MGNkLWI3MTktNjA1Yzg1YTRiMDA0Il0sWyIhPSIsWyJmaWVsZCIsNjQzLG51bGxdLFsiZmllbGQiLDM3NCxudWxsXV1dfSwiZGF0YWJhc2UiOjJ9LCJkaXNwbGF5IjoidGFibGUiLCJ2aXN1YWxpemF0aW9uX3NldHRpbmdzIjp7fSwib3JpZ2luYWxfY2FyZF9pZCI6bnVsbH0=) though